docs/design/secrets.md

```diff
@@ -81,8 +86,8 @@ Collecting remaining TODOs here:
 1.  Describe the lifecycle of secrets
 2.  Determine how/whether secrets interact with service accounts; are namespaces enough for the
     security scope? Can/should my security context affect what secrets I have access to?
-3.  How will pods know when secret values change?  Should there be a way to express that a pod
-    which consumes a secret should be restarted when the secret changes?
+3.  How will containers know when secret values change?  Should there be a way to express that a
```

Comment:
I agree that secrets should be immutable from the standpoint of a running container; I'll add some more context to this so that's clear.